.compute

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+export BUILD_ARCH=x86-64
+
+# Install MKL for CPU version
+wget https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS-2019.PUB
+apt-get update \
+    && apt-get install -y apt-transport-https \
+    && apt-key add GPG-PUB-KEY-INTEL-SW-PRODUCTS-2019.PUB \
+    && echo deb https://apt.repos.intel.com/mkl all main \
+       >/etc/apt/sources.list.d/intel-mkl.list \
+    && apt-get update
+apt-get install -y intel-mkl-64bit-2018.2-046
+
+# Install cmake
+wget https://github.com/Kitware/CMake/releases/download/v3.12.2/cmake-3.12.2-Linux-x86_64.sh
+chmod +x cmake-3.12.2-Linux-x86_64.sh
+./cmake-3.12.2-Linux-x86_64.sh --skip-license --prefix=/usr/local
+
+# Install other mandatory packages
+apt-get install -y \
+    g++ \
+    git \
+    libev++-dev \
+    libssl-dev \
+    libboost-all-dev \
+    libtool \
+    libgoogle-perftools-dev \
+    libpcre3-dev \
+    libprotobuf-dev \
+    protobuf-compiler \
+    python3-pip \
+    unzip \
+    wget
+
+# Build the server 
```

Comment:
I don't think this comment is correct.

COMPILE_SERVER defaults to OFF ans it's not set to ON here.

Reply:
Agreed. Changed it in the next commit.